docs/user/04-metrics.md

```diff
@@ -412,26 +412,7 @@ spec:
       endpoint:
         value: https://backend.example.com:4317
 ```
-Note that the metrics from system Namespaces are dropped by default for the `prometheus`, `runtime`, and `otlp` inputs. However, the metrics from system Namespaces are collected by default for the `istio` input.
-
-The following example collects runtime metrics from all Namespaces including system Namespaces:
-```yaml
-apiVersion: telemetry.kyma-project.io/v1alpha1
-kind: MetricPipeline
-metadata:
-  name: backend
-spec:
-  input:
-    runtime:
-      enabled: true
-      namespaces:
-        system: true
-  output:
-    otlp:
-      endpoint:
-        value: https://backend.example.com:4317
-```
-
+Note that the metrics from system Namespaces are excluded by default for the `prometheus`, `runtime`, and `otlp` inputs. However, the metrics from system Namespaces are included by default for the `istio` input.
```

Comment:
[guidelines are changing](https://github.com/kyma-project/kyma/issues/18282#issuecomment-1846626347); "namespace" will be lower case

```suggestion
Note that the metrics from system namespaces are excluded by default for the `prometheus`, `runtime`, and `otlp` inputs. However, the metrics from system namespaces are included by default for the `istio` input.
```

Reply:
Thanks, good to know! I also improved the wording a bit